extensions/ql-vscode/src/model-editor/generate.ts

```diff
@@ -5,19 +5,15 @@ import type { QueryRunner } from "../query-server";
 import type { CodeQLCliServer } from "../codeql-cli/cli";
 import type { ProgressCallback } from "../common/vscode/progress";
 import { getOnDiskWorkspaceFolders } from "../common/vscode/workspace-folders";
-import type { ModeledMethod } from "./modeled-method";
 import { runQuery } from "../local-queries/run-query";
 import type { QueryConstraints } from "../local-queries";
 import { resolveQueries } from "../local-queries";
 import type { DecodedBqrs } from "../common/bqrs-cli-types";
+
 type GenerateQueriesOptions = {
   queryConstraints: QueryConstraints;
   filterQueries?: (queryPath: string) => boolean;
-  parseResults: (
-    queryPath: string,
-    results: DecodedBqrs,
-  ) => ModeledMethod[] | Promise<ModeledMethod[]>;
```

Comment:
Why are we getting rid of the `parseResults` option? 🤔

Reply:
`parseResults` was very specific to producing `ModeledMethod[]`, while that's not something we want to do for this type model generation (we can just put the rows as-is into a YAML file). The only thing that `parseResults` was used for was `await onResults(await parseResults(queryPath, bqrs))`, which can very easily be replaced by `await onResults(queryPath, bqrs)` with very minor changes on the consumer side.